tools/codegen/cmd/crd_manifest_merge.go

```diff
@@ -30,6 +33,8 @@ var crdManifestMerge = &cobra.Command{
 
 func init() {
 	rootCmd.AddCommand(crdManifestMerge)
+
+	rootCmd.PersistentFlags().StringVar(&manifestMergePayloadManifestPath, "manifest-merge:payload-manifest-path", manifestmerge.DefaultPayloadFeatureGatePath, "path to locate the featuregate YAML.")
```

Comment:
Since this is a dir expecting multiple files, may want to make it clearer that this is a dir